cunumeric/array.py

```diff
@@ -4014,7 +4014,7 @@ def find_common_type(*args: Any) -> np.dtype[Any]:
                 scalar_types.append(array.dtype)
             else:
                 array_types.append(array.dtype)
-        return np.find_common_type(array_types, scalar_types)  # type: ignore
+        return np.find_common_type(array_types, scalar_types)
```

Comment:
I'd make sure this change doesn't make mypy unhappy.

Reply:
@this is from #980. I'll let @ipdemes merge the changes in `putmask`

I think this change was needed to make sure mypy doesn't complain, so it does it make it happy. (@ipdemes would know more)

Reply:
Yes, mypy was complaining about this `type: ignore` so I removed it

Reply:
Ha, mypy in CI complaining now. I will put it back